src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -206,6 +202,23 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine($"#iprangefiltering=<0 or 1>");
         }
 
+        private int CalculateBanTime(Network network)
+        {
+            int banTime = 0;
+
+            // Ensure that the network's MaxReorgLength always has a value.
+            int maxReorgLength = network.Consensus.MaxReorgLength == 0 ? Network.FallBackMaxReorg : (int)network.Consensus.MaxReorgLength;
+
+            // If the network's consensus options is a subclass of ConsensusOptions then it is not the base
+            // and therefore we conclude that the network is NOT Proof-Of-Work.
+            if (network.Consensus.Options.GetType().IsSubclassOf(typeof(ConsensusOptions)) || network.Consensus.Options.GetType() != typeof(ConsensusOptions))
```

Comment:
Can we avoid this logic by having a single variable called `TargetSpacingSeconds` - i.e. rename `PowTargetSpacing` to `TargetSpacingSeconds`?

Reply:
I suppose `network.Consensus.IsProofOfStake` isn't of use here?